articles/libraries/_includes/_get_lock_latest_version.md

```diff
@@ -0,0 +1,23 @@
+### Update Lock
+
+Update the Lock library using npm or yarn.
+
+```bash
+# installation with npm
+npm install --save auth0-lock
+
+# installation with yarn
+yarn add auth0-lock
+```
+
+Once updated, you can add it to your build system or bring it in to your project with a script tag.
+
+```html
+<script type="text/javascript" src="node_modules/auth0-lock/build/lock.js"></script>
+```
+
+If you do not want to use a package manager, you can retrieve Lock from Auth0's CDN.
+
+```html
+<script src="${lock_url}"></script>
+```
```

Comment:
Careful, this still points to Lock v10.20

Reply:
We'll update the variable to Lock11 when we ship

Reply:
Yes, this variable will be updated as soon as we have a release version to list